main.cpp

```diff
@@ -401,28 +501,33 @@ Thing 7) seats
  */
 struct Seats
 {
-    //number of seats
+    
     int numberSeats = 4;
-    //seat width
     int seatWidth = 30;
-    //seat recline angle
     float reclineAngle = 44.3f;
-    //seat heating
     bool seatHeating = true;
-    //seat forward movement
     int forwardMovement = 22;
 
-    //seat driver
-    void seatDriver();
-    //heat passenger
-    void heatPassangerSeat();
-    //adjust position
-    void adjustSeatPosition();
 
+    void seatDriver(bool numberOfDrivers);
+    void heatPassangerSeat(int temperature);
+    void adjustSeatPosition(bool degree);
 };
 
+void Seats::seatDriver(bool numberOfDrivers)
+    {
+    numberOfDrivers = true;
+    }
 
+void Seats::heatPassangerSeat(int temperature)
+    {
+    temperature = 38;
+    }
 
+void Seats::adjustSeatPosition(bool degree)
+    {
+    degree = true;  
+    }
```

Comment:
Fix this according to my previous pull request review instructions
- [ ] revise